templates/workloads/partials/cf/publish.yml

```
@@ -0,0 +1,31 @@
+{{- if .Publish}}
+{{- range $topic := .Publish.Topics}}
+{{$topic.Name}}SNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: ${AWS::StackName}-{{$topic.Name}}-SNSTopic
```

Can we remove the suffix from the topic? I think it'd be redundant since the resource itself is an `SNS::Topic`
```suggestion
      TopicName: ${AWS::StackName}-{{$topic.Name}}
```

---

Gotcha! Adjusted both.